### PR TITLE
feat(tslint-config): Add package tslint-config.

### PR DESCRIPTION
**Pull request by lipis**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/tslint-config/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+yarn add --dev @wireapp/tslint-config ^
+               tslint-config-prettier ^
+               tslint-plugin-prettier ^
+               tslint-react-hooks ^
+               tslint-react ^
+               prettier ^
+               tslint
+```
+
+**Edit `.eslintrc.json`**:
```
Comment (ffflorian) [inline packages/tslint-config/README.md]:
```suggestion
**Edit `tslint.json`**:
```

File: packages/tslint-config/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+```jsonc
+{
+  // ...
+  "extends": "@wireapp/eslint-config"
```
Comment (ffflorian) [inline packages/tslint-config/README.md]:
```suggestion
  "extends": "@wireapp/tslint-config"
```

File: packages/tslint-config/README.md (hunk 1 above)
Comment (bennyn) [inline packages/tslint-config/README.md]: eslint? 🤔

File: packages/tslint-config/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "peerDependencies": {
+    "prettier": ">=1",
+    "tslint-config-prettier": ">=1",
+    "tslint-plugin-prettier": ">=2",
+    "tslint-react-hooks": ">=2",
+    "tslint-react": ">=4",
+    "tslint": ">=5"
+  },
+  "repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/tslint-config",
+  "version": "0.0.3"
```
Comment (ffflorian) [inline packages/tslint-config/package.json]:
```suggestion
  "version": "0.0.1"
```

File: packages/tslint-config/package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Wire TSLint config",
+  "license": "GPL-3.0",
+  "main": "tslint.json",
+  "name": "@wireapp/tslint-config",
+  "peerDependencies": {
+    "prettier": ">=1",
```
Comment (ffflorian) [inline packages/tslint-config/package.json]:
```suggestion
    "prettier": ">= 1",
```
same for the other peer dependencies

File: tslint.json [hunk 5]
```diff
@@ -1,41 +1,5 @@
...
-    "prefer-object-spread": true,
-    "prefer-readonly": true,
-    "prefer-template": true,
-    "prettier": true,
-    "react-hooks-nesting": "error",
-    "space-within-parens": [true, 0]
-  },
+  "extends": ["@wireapp/tslint-config"],
   "linterOptions": {
     "exclude": ["**/bower_components/**", "**/dist/**", "**/node_modules/**"]
```
Comment (bennyn) [inline tslint.json]: Let's add `"**/*.d.ts"` here, like in: https://github.com/wireapp/wire-desktop-packages/blob/%40wireapp/desktop-updater%400.2.7/tslint.json#L36

File: packages/tslint-config/package.json (hunk 3 above)
Comment (lipis) [reply packages/tslint-config/package.json]: I already published it and use it in this project :) It was easier than yarn link

**Comment by ffflorian** [comment]
Please rename to `feat(tslint-config): Add package tslint-config`.

File: packages/eslint-config/package.json [hunk 6]
```diff
@@ -4,13 +4,13 @@
...
-    "eslint-plugin-prettier": ">=3",
-    "eslint-plugin-react": ">=7",
-    "prettier": ">=1.17"
+    "eslint": ">= 5",
+    "eslint-config-prettier": ">= 4",
+    "eslint-plugin-jsdoc": ">= 4",
+    "eslint-plugin-no-unsanitized": ">= 3",
+    "eslint-plugin-prettier": ">= 3",
+    "eslint-plugin-react": ">= 7",
+    "prettier": ">= 1.17"
```
Comment (lipis) [inline packages/eslint-config/package.json]: @ffflorian on a second thought.. I think I prefer without the space, usually there are no spaces here "^12.1.0", "~1.2.3", ">=1.23"

Comment (ffflorian) [reply packages/eslint-config/package.json]: Ah, you are right. That's also what [npm prefers](https://docs.npmjs.com/misc/semver). You can revert it.

**Comment by ffflorian** [comment]
@lipis merge it! :rocket:

**Pull request merged**